windows.md

```diff
@@ -0,0 +1,29 @@
+Windows Node Support
```

Comment:
title formatting? also should we call this "Windows Support" or even generalize to "Supported Operating Systems"?

Reply:
I think this is specifically about windows support since we have deployment instructions below? Formatting updated though.

Reply:
Understood, just wondering if users wouldn't benefit from a broader resource, e.g. what OS's does Kubecost support.